### PR TITLE
Package as Flatpak

### LINKED ISSUE #59: Package as Flatpak
This seems like a great example of an app that should be sandboxed, actually

### PR DESCRIPTION
**Pull request by zeebok**
Fixes #59

### CODE REVIEW

**Comment by zeebok** [comment]
First time I have flatpak'd something, so let me know if there are other files for the backend that I need to update

**Review by danrabbit — CHANGES_REQUESTED** [review]
Hey right on, thanks for tackling this!

If this is ready for review, we can go ahead and remove the comments here in the flatpak.yml since those are all part of the tutorial in the docs and not really necessary here in production

and then we'll want to build the Flatpak in CI, so you can replace the contents of .github/workflows/main.yml with something like this: https://github.com/elementary/screenshot/blob/master/.github/workflows/ci.yml

Just make sure to change out `screenshot` for `capnet-assist` and that should be all you need to do to get it building

**Comment by zeebok** [comment]
Thanks for the comments and pointing to the CI file. This should be all set. Is there a specific group I should request to review this?

**Comment by danrabbit** [comment]
Awesome thanks!

It doesn't hurt to request `desktop developers` but I can go ahead and re-review for you :)

File: io.elementary.capnet-assist.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  - '--socket=wayland'
+  - '--share=network'
+
+modules:
+  - name: gcr
+    buildsystem: meson
+    sources:
+        - type: git
+          url: https://gitlab.gnome.org/GNOME/gcr.git
+          tag: '3.36'
```
Comment (Marukesu) [inline io.elementary.capnet-assist.yml:21]: The Gnome SDK already include gcr, no need to build it here too.

File: io.elementary.capnet-assist.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+sdk: io.elementary.Sdk
+
+command: io.elementary.capnet-assist
+
+finish-args:
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--share=network'
+
```
Comment (danrabbit) [inline io.elementary.capnet-assist.yml:14]: don't forget the sandbox hole for the dark style preference :)
```suggestion
  - '--system-talk-name=org.freedesktop.Accounts'
```

**Review by danrabbit — CHANGES_REQUESTED** [review]
Ah noticed a small problem in the manifest

File: io.elementary.capnet-assist.yml (hunk 1 above)
Comment (zeebok) [reply io.elementary.capnet-assist.yml:21]: Hmm alright, when I didn't include it as a module I would consistently get warnings surrounding gcr when I ran tests. I will go back and see if it is caused by something else!

File: io.elementary.capnet-assist.yml (hunk 2 above)
Comment (zeebok) [reply io.elementary.capnet-assist.yml]: Oh man, is this mentioned in the developer guide and I totally missed it?

Comment (danrabbit) [reply io.elementary.capnet-assist.yml]: I actually just added a section for it the other day! So it probably wasn't when you pushed this

**Review by danrabbit — APPROVED** [review]
Awesome, it looks like this builds and runs fine to me! I'm happy to get this merged and then we can continue to test

nice work!

**Pull request merged**